lib/osf-components/addon/components/sharing-icons/popover/template.hbs

```diff
@@ -8,11 +8,10 @@
     ...attributes
 >
     <FaIcon @icon='share-alt' />
-    <BsPopover
+    <EmberPopover
         local-class='Popover'
-        @placement='left'
-        @viewportSelector='#{{@parentId}}'
-        @autoPlacement={{true}}
+        @side='left'
+        @popperContainer='#{{@parentId}}'
         @renderInPlace={{true}}
```

Comment:
It doesn't look like EmberPopover supports `renderInPlace` so this can be removed